3rdparty/udis86/libudis86/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# UJIT: CMake file for udis86 to avoid dependencies on autoconf and libtool.
+# UJIT: Based on https://github.com/vmt/udis86/pull/99/files by hasherezade.
+# UJIT: Copyright (C) 2020 Anton Soldatov.
+
+set(ITAB_H "${CMAKE_CURRENT_LIST_DIR}/itab.h")
+set(ITAB_C "${CMAKE_CURRENT_LIST_DIR}/itab.c")
+
+if(NOT EXISTS ${ITAB_H} OR NOT EXISTS ${ITAB_C})
+  message(FATAL_ERROR "You must generate files: ${ITAB_H}, ${ITAB_C} before you start.")
+endif()
+
+set(SOURCES_LIBUDIS86
```

Comment:
Did you try out of source build? I'm not 100% sure on using relative paths there and how CMake might work with them in different scenarios. Foolproof way to me seems to use "${SOURCE_DIR}" which was previously obtained in parent CMakeLists.txt... So something like this:


```cmake
# in 3rdparty/CMakeLists.txt
ExternalProject_Get_Property(udis86_ext SOURCE_DIR)
set(UDIS86_SOURCE_DIR "${SOURCE_DIR}")
add_subdirectory(udis86)
```

and then use this:

```cmake
set(SOURCES_LIBUDIS86
   "${UDIS86_SOURCE_DIR}/decode.c"
   ... #etc
)
```

Reply:
Out of source build is the one used in Travis CI, everything passes there. Anyway, the patch is reworked.